physical/physical_test.go

```diff
@@ -283,7 +283,7 @@ func testBackend_ListPrefix(t *testing.T, b Backend) {
 
 func testHABackend(t *testing.T, b HABackend, b2 HABackend) {
 	// Get the lock
-	lock, err := b.LockWith("foo", "bar")
+	lock, err := b.LockWith("/core/lock/foo", "bar")
```

Comment:
not the backend. the previous test (testBackend_ListPrefix) left some keys. i can change testBackend_ListPrefix to make it cleanup stuff.

I thought we'd better simulate the true behavior. You mentioned that vault will do locking under namespace /core/lock.